src/topology/algebra/ordered.lean

```diff
@@ -995,6 +1034,84 @@ lemma cinfi_of_cinfi_of_monotone_of_continuous {f : α → β} {g : γ → α}
 by rw [infi, cInf_of_cInf_of_monotone_of_continuous Mf Cf
   (λ h, range_eq_empty.1 h ‹_›) H, ← range_comp]; refl
 
+section densely_ordered
+
+variables [densely_ordered α] {a b : α}
+
+lemma is_connected_Icc : is_connected (Icc a b) :=
+is_connected_closed_iff.2
+begin
+  rintros s t hs ht hab ⟨x, hx⟩ ⟨y, hy⟩,
+  wlog hxy : x ≤ y := le_total x y using [x y s t, y x t s],
+
```

Comment:
Can you remove the empty lines, here and below?

Reply:
Done.